modules/model/item/components/physical.js

```diff
@@ -68,6 +68,22 @@ export class PhysicalItemModel extends BaseItemModel
         return enc
     }
 
+    /**
+     * Reduces the quantity of this Item by specified amount.
+     *
+     * @param {number} amount by how much should the quantity be reduced?
+     *
+     * @returns {Promise<ItemWfrp4e|boolean>}
+     */
+    async reduceQuantity(amount = 1) {
+        const newQuantity = Math.max(0, this.quantity.value - amount);
+
+        if (newQuantity === this.quantity.value)
+            return false;
+
+        return await this.parent.update({"system.quantity.value": newQuantity})
+    }
+
```

Comment:
It's worth noting that since the DataModel defines the minimum to be 0, you could simply update it with `system.quantity.value - amount` and it should handle it fine, I'm not sure if that's "recommended" though.

Reply:
Right, it would handle it. The only reason I did that was to not trigger update if item is already at 0 quantity, but now I tested it and it handles that too 😅